needy/utility.py

```diff
@@ -0,0 +1,6 @@
+class DummyContextManager():
+    def __enter__(self):
+        return None
+
+    def __exit__(self, exc_type, exc_value, traceback):
+        return False
```

Comment:
IIRC, `return False` is the same as `pass` in the case of the `__exit__` method.